src/main/java/frc/robot/commands/IntakeControl.java

```diff
@@ -55,6 +55,6 @@ public boolean isFinished() {
       // return intakeSubsystem.isSensor7True() == false;
       return false;
     else
-      return intakeSubsystem.isSensor7True() == false;
+      return intakeSubsystem.isBackSensorActive() == false;
```

Comment:
suggest to add a timer here, in case the sensor has issue and we shouldn't be stuck here.

Reply:
There is already a timer on this but its done elsewhere with the command using .withTimeout(0.3)